app/view/button/StreetViewTool.js

```diff
@@ -33,7 +33,14 @@ Ext.define('CpsiMapview.view.button.StreetViewTool', {
     name: 'streetViewToolButton',
 
     /**
-     * Enable toogle mode for the button.
+     * The tooltip to display when hovering over the button
+     *
+     * @private
+     */
+    tooltip: 'Activate the StreeView tool',
```

Comment:
Typo: Stree**t**

Reply:
Good catch, was too early for me, I guess :wink:

Reply:
Was too late at night for me! Thanks!

Reply:
😅